src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -127,7 +129,7 @@ function LHNOptionsList({
                 keyExtractor={keyExtractor}
                 renderItem={renderItem}
                 estimatedItemSize={optionMode === CONST.OPTION_MODE.COMPACT ? variables.optionRowHeightCompact : variables.optionRowHeight}
-                extraData={[currentReportID]}
+                extraData={extraData}
```

Comment:
can you please add context here? What bug was there when use `currentReportID` only?

Reply:
+1 this deserves a comment

Reply:
Hey this was actually added [here](https://github.com/Expensify/App/pull/32336). But I have added the comments in the code in this PR.

Reply:
so `renderItem` depends on these values `reportActions, reports, policy, personalDetails` and just relying on `data` might cause re-render of the list when re-rendering is necessary?